README.md

```diff
@@ -1,20 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 
-# A Portfolio Project
+# My Portfolio
 > Description the project.
 
```

Comment:
[OPTIONAL] Kindly consider adding a Get start section with the instruction on how to use the App/Project. This is going to be useful for the people who check your projects and wants to give them a try!